README.md

```diff
@@ -62,6 +62,27 @@ The JavaScript interface currently only supports IQL-strict queries.
 
 ### Development
 
+#### Shadow-cljs
+
+To develop with a cljs environment, use shadow-cljs. The simplest way is to use a bare Node-based REPL. 
+
+From the command-line, run:
+
+```shell 
+npx shadow-cljs server # start the shadow-cljs server
+npx shadow-cljs node-repl # start the node repl
```

Comment:
The `node-repl` command also (implicitly) starts a server, so we could remove the first command. (There is a small difference, in that if you start the server manually, it remains up even if you close the REPL process)

Reply:
Yes, I was relying on that difference to support editor REPLs. I mention that in the lines right below, but hmm, maybe it would be clearer to provide two sets of instrs, one for CLI, one for editors.

Reply:
OK, updated to be a bit clearer, hopefully.

Reply:
Ah. It should be the same for editors either way - if you've run `shadow-cljs node-repl`, you can connect to it with editor via `nrepl`. `(shadow/node-repl)` will connect to the existing node-repl if it's running, or start a new one. (So your terminal + editor will share the same REPL environment.)

Reply:
I know, but that's incidental to the usual intent of running node-repl. If someone's new to clojure/script, I don't want to confuse them by also starting a repl that I then want them to _not_ use in the next instructions.  

I figure, one less thing in their mental model, and cljs repls are already extra-convoluted compared to clj.